guides/Tong/README.md

```diff
@@ -0,0 +1,173 @@
+  
+  ![](lanEm0.png)
+   
+   When the screen asks, `Do VLANs need to be set up first?`, that is referring to setting up the virtual LAN. We have already done that in previous steps, so type `n` for 'no' and hit `Enter`.
+   
+   The first interface PFsense asks you to assign is the WAN. In previous steps we associated the WAN with Network Card/Adapter 4 (2 on Mac) under the PFsense Adapter 3 tab in VirtualBox. This tab basically refers to em2, so type `em2` and hit `Enter` to answer the question.
+      
+   The second interface PFsense asks you to assign is the LAN. In previous steps we associated the LAN with Network Card/Adapter 2  (0 on Mac) under the PFsense Adapter 1 tab in VirtualBox. This tab basically refers to em0, so type `em0` and hit `Enter` to answer the question.
+   
+   The third interface PFsense asks you to assign is an optional interface. Here, we have the DMZ that we setup earlier. We associated the DMZ with Netword Card/Adapter 2 (1 on Mac) under the PFsense Adapter 2 tab in VirtualBox. This tab basically refers to em1, so type `em1` and hit `Enter` to answer the question.
+   
+   The fourth interface PFsense asks you to assign is another optional interface. We don't have another one here, so just hit `Enter` to move on.
+   
+   Check the image below to be sure you have the following commands typed.
+   
+   ![](enterInterfaces.PNG)
+
+2. Set the interfaces' IP addresses
+
+   When you return to the welcome screen, enter `2` to select option 2. Here, we will be setting the IP addresses of the interfaces we assigned within PFsense. You can set the IP address for whichever one you want to do first. The order doesn't matter. Let's start with the WAN. Type `1` to select WAN.
+
+   ![](option2.PNG)
+   
+   When the screen asks for the IP address for the WAN, type `172.31.1.2`. When it asks for DHCP, type `n`.
+   
+   Make sure the following commands you type are the same as shown in the image below:
+   
+   ![](wanConfigure.PNG)
+   
+   Next, we will configure the LAN. When the screen asks for the IP address for the LAN, type `172.20.241.254`. Type `24` for its IPv4 submask. When it asks for DHCP, type `n`. Make sure the rest of your commands are as follows:
+   
+   ![](lanConfigure.PNG)
+   
+   Next, we will configure the DMZ. Type `172.20.240.254` for its IP address. Then Type `n` for DHCP. Make sure the rest of your commands are as follows:
+   
+   ![](dmzConfigure.PNG)
+   
+   Congrats! You have configured the PFsense VM! (on the command line)
+   
+   ![](aftermath.PNG)
+   
+   Next, we will finish the PFsense configuration on the Windows 10 user interface. Reason being, the rest of the configuration steps will be much more difficult to do on the command line. Keep PFsense running and move onto the Windows 10 setup guide (coming soon). 
+   
```

Comment:
You will link to the windows guide if it gets made. Though you should still include windows logging into the pfsense web interface here.